### ISSUE TITLE
Allow remote access by default

### ISSUE THREAD

**Issue by mattias-p** [zonemaster/zonemaster-gui#204]
We should allow remote access to zm-rpcapi by default. This would make installation testing easier.

On CentOS there are two things that need to be updated. 
1. zm-rpcapi should listen to 0.0.0.0 by default, not 127.0.0.1 as it is now.
2. The port 5000/tcp needs to be opened up in the firewall.

**Comment by matsduf** [comment]
I agree that testing will be easier, but I still think it is a good practice to keep it to localhost only. In FreeBSD it is easy to open it for another IP address, or 0.0.0.0, by adding a line to /etc/rc.conf. Is there a standard for configuration of start script in CentOS?
